new.go

```diff
@@ -244,7 +245,12 @@ func newBuilder(ctx context.Context, store storage.Store, options BuilderOptions
 		options.FromImage = ""
 	}
 
-	systemContext := getSystemContext(options.SystemContext, options.SignaturePolicyPath)
+	systemContext := getSystemContext(store, options.SystemContext, options.SignaturePolicyPath)
+	if options.BlobDirectory != "" {
+		systemContext.BlobInfoCacheDir = options.BlobDirectory
```

Comment:
This seems to confuse the buildah blob cache and the c/image blob _info_ cache.

Reply:
Removing in new PR>